designer/test/i18nForTest.js

```diff
@@ -0,0 +1,23 @@
+import { i18n } from "../client/i18n";
```

Comment:
you don't need this file. use these imports
```import { i18n, initI18n } from "../client/i18n";```
call `initI18n()` at the top of your test file

Reply:
somehow that didn't work in the test , so i added this to fully initialize i18n